Cargo.toml

```diff
@@ -14,6 +14,8 @@ libc = "0.2"
 aligned_alloc = "0.1"
 num-complex = { version = "0.1.40", default-features = false }
 tensorflow-sys = { version = "0.12.0", path = "tensorflow-sys" }
+byteorder = "*"
```

Comment:
I noticed you have Cargo.lock in .gitignore; would you like me to reify the version number here instead?

(Also, out of curiosity, why not use the Cargo.lock mechanism?)

Reply:
This crate is a library, and in accordance to the [Cargo docs](https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html), libraries don't have a Cargo.lock. I would strongly recommend these new dependencies to have a more appropriate version range.

Reply:
We definitely need minimum versions for the dependencies.

Reply:
done